WordPress/Tests/Theme/FileIncludeUnitTest.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+/* ---- include or require ----
+ * No bad or good here, just issue a warning for manual check.
+ * */
+include('/file.ext');
+include_once('/file.ext');
+require('/file.ext');
+require_once('/file.ext');
```

Comment:
`include` and `require` are language constructs and do not need parenthesis, so it might be good to have some examples with more syntax variations:

```php
include dirname( __FILE__ ) . '/myfile.php';
require_once $path_to_file;
```